pallets/loans/src/benchmarking.rs

```diff
@@ -85,11 +85,9 @@ fn config_mocks() {
 	MockPrices::mock_feed_value(|_, _, _| Ok(()));
 	MockPrices::mock_register_id(|_, _| Ok(()));
 	MockPrices::mock_collection(|_| MockDataCollection::new(|_| Ok(Default::default())));
-	MockChangeGuard::mock_note(|_, _| Ok(sp_core::H256::default()));
-	MockChangeGuard::mock_released(|_, _| {
-		Ok(ChangeOf::<Runtime>::Policy(
-			Helper::<Runtime>::create_policy(),
-		))
+	MockChangeGuard::mock_note(|_, change| {
+		MockChangeGuard::mock_released(move |_, _| Ok(change.clone()));
+		Ok(sp_core::H256::default())
 	});
```

Comment:
This is a new feature of `mock-builder` that I needed for testing this part.

Now you can define mocks inside mock executions. It unlocks the insert-get mock pattern, meaning you can return what you've previously inserted in another mock. In other words, you will always get the last insertion.

```rust
MyMock::mock_insert(|value| MyMock::mock_get(move || value));
```

cc @mustermeiszer @cdamian @NunoAlexandre @wischli

Reply:
Moved to its own PR: https://github.com/centrifuge/centrifuge-chain/pull/1420